msupply-foundation/open-msupply
# 3498 pt 1 inventory adjustment permissions

Part of #3498

# 👩🏻‍💻 What does this PR do? 
 

Adds permissions mappings for inventory adjustments.

# 🧪 How has/should this change been tested? 


- Disable the `Enter Inventory Adjustments`, `Edit Inventory Adjustments` and `Finalise Inventory Adjustments` permissions in mSupply for your omSupply user
- Log out and log in again
- Try to do an inventory adjustment via the stock view modal
- [ ] You should get a permission denied error
- Enable one of the `Enter Inventory Adjustments` permission
-  Log out and log in again
- Try to do an inventory adjustment via the stock view modal
- [ ] It should not error 🎉 

Query permissions not yet used anywhere to test from UI...

## 💌 Any notes for the reviewer?


There are 3 IA mutation related permissions in OG - `Enter Inventory Adjustments`, `Edit Inventory Adjustments` and `Finalise Inventory Adjustments`.

However, my understanding with omSupply inv adjustments is that we'll finalise them immediately - i.e. create only. I've mapped all of those 3 permissions to the one OMS `MutateInventoryAdjustment` permission - is that correct?

## 📃 Documentation

_No user facing changes_
